commit b47f7b38452e20cf02f417af100c434e75713e58
Merge: 63d287c 827f93e
Author: tomc <tomc@cern.ch>
Date:   Wed Jan 16 12:47:33 2019 +0100

    fixed conflicts

--------------------------------------------------------------------------------------------------

Running test: Run2018-17Sep2018 --> FAILED
Output:   ----- Begin Fatal Exception 16-Jan-2019 12:49:02 CET-----------------------
   An exception of category 'ConfigFileReadError' occurred while
      [0] Processing the python configuration file named ../multilep.py
   Exception Message:
   python encountered the error: <type 'exceptions.SyntaxError'>
   invalid syntax (multilep.py, line 142)
   ----- End Fatal Exception -------------------------------------------------

--------------------------------------------------------------------------------------------------

Running test: Run2018-PromptReco --> FAILED
Output:   ----- Begin Fatal Exception 16-Jan-2019 12:49:03 CET-----------------------
   An exception of category 'ConfigFileReadError' occurred while
      [0] Processing the python configuration file named ../multilep.py
   Exception Message:
   python encountered the error: <type 'exceptions.SyntaxError'>
   invalid syntax (multilep.py, line 142)
   ----- End Fatal Exception -------------------------------------------------

--------------------------------------------------------------------------------------------------

Running test: Run2017-31March2017 --> FAILED
Output:   ----- Begin Fatal Exception 16-Jan-2019 12:49:04 CET-----------------------
   An exception of category 'ConfigFileReadError' occurred while
      [0] Processing the python configuration file named ../multilep.py
   Exception Message:
   python encountered the error: <type 'exceptions.SyntaxError'>
   invalid syntax (multilep.py, line 142)
   ----- End Fatal Exception -------------------------------------------------

--------------------------------------------------------------------------------------------------

Running test: Run2016-17Jul2018 --> FAILED
Output:   ----- Begin Fatal Exception 16-Jan-2019 12:49:04 CET-----------------------
   An exception of category 'ConfigFileReadError' occurred while
      [0] Processing the python configuration file named ../multilep.py
   Exception Message:
   python encountered the error: <type 'exceptions.SyntaxError'>
   invalid syntax (multilep.py, line 142)
   ----- End Fatal Exception -------------------------------------------------

--------------------------------------------------------------------------------------------------

Running test: Autumn18MiniAOD --> FAILED
Output:   ----- Begin Fatal Exception 16-Jan-2019 12:49:05 CET-----------------------
   An exception of category 'ConfigFileReadError' occurred while
      [0] Processing the python configuration file named ../multilep.py
   Exception Message:
   python encountered the error: <type 'exceptions.SyntaxError'>
   invalid syntax (multilep.py, line 142)
   ----- End Fatal Exception -------------------------------------------------

--------------------------------------------------------------------------------------------------

Running test: Fall17MiniAODv2 --> FAILED
Output:   ----- Begin Fatal Exception 16-Jan-2019 12:49:06 CET-----------------------
   An exception of category 'ConfigFileReadError' occurred while
      [0] Processing the python configuration file named ../multilep.py
   Exception Message:
   python encountered the error: <type 'exceptions.SyntaxError'>
   invalid syntax (multilep.py, line 142)
   ----- End Fatal Exception -------------------------------------------------

--------------------------------------------------------------------------------------------------

Running test: Summer16MiniAODv3 --> FAILED
Output:   ----- Begin Fatal Exception 16-Jan-2019 12:49:07 CET-----------------------
   An exception of category 'ConfigFileReadError' occurred while
      [0] Processing the python configuration file named ../multilep.py
   Exception Message:
   python encountered the error: <type 'exceptions.SyntaxError'>
   invalid syntax (multilep.py, line 142)
   ----- End Fatal Exception -------------------------------------------------
